Utils/layouts/head.js

```diff
@@ -27,6 +27,7 @@ const AppHead = ({ path, t }) => {
         type="image/x-icon"
         href="/static/favicon.ico"
       ></link>
+      <link rel="stylesheet" href="/static/accordion.css" />
```

Comment:
Just wondering: aren't these styles provided by the Library? I'm just a little surprised to see them loaded here is all.

Reply:
> > They are, apart from the extra ones just added in the comment section which I am working one right now. For the accordion to work without JS, we will have to create another ticket. @andreferreiradlw and I already discussed that.
> 
> OK, so
> 
> * should we wait for the stuff in the comments to be done before approving this PR or will it be a separate card?
> * on the accordion: that's fine for this PR.

Yeah. Done.